### PR TITLE
Add initial FAQ.

### PR DESCRIPTION
**Pull request by kmoscoe**
Answer questions from one.org, Tech Soup, and other partners.

### CODE REVIEW

**Comment by keyurva** [comment]
Added @pradh and @shifucun to the reviewers list. We should get these verified / corrected by them before publishing.

File: custom_dc/faq.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+No. At this time, there is no plan to use any Google-specific ML technologies for NL processing in custom Data Commons. All models are open-source.
+
+### Where does the ML model run and where are embeddings stored?
+
+The ML model runs entirely on your custom Data Commons instance, inside the Docker image. It does not use any Google-hosted systems, and data is never leaked to the main Data Commons. If a natural-language query requires data to be joined from the main Data Commons site, the custom site will use the embeddings that are locally generated before making the call to the main Data Commons to fetch the data.
+
+### Does the model use feedback from user behavior to adjust scoring?
+
+No. At this time, there is no plan to enable user feedback mechnanisms for custom Data Commons.
```
Comment (kmoscoe) [inline custom_dc/faq.md:42]:
```suggestion
No. At this time, there is no plan to enable user feedback mechanisms for custom Data Commons.
```

**Comment by kmoscoe** [comment]
Ping, Bo and Prashanth?

File: custom_dc/faq.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+### Will my custom data end up in the main Data Commons database?
+
+No. Even if a query needs data from the main Data Commons, gleaned from the (incomplete) results from your custom data, it will never transfer any custom data to the main site.
+
+## Natural language processing
+
+### How does the natural language interface work?
+
+The NL interface uses an open-source Python ML library, Sentence Transformers model, from [https://huggingface.co/sentence-transformers](https://huggingface.co/sentence-transformers). When you load data into a custom instance, the Data Commons NL server generates embeddings for both the main Data Commons data, and for your custom data, based on the statistical variables and search descriptions you have defined in your configuration. When a query comes in, the server generates equivalent embeddings, and the variables are assigned a relevance score based on cosine similarity.
```
Comment (pradh) [inline custom_dc/faq.md:30]: What do you think about something like (of course, needs more polish):

The NL interface has the ability to use a combination of different embedding models, heuristics and LLMs (as fallback).  Given an NL query, it first detects schema (variables, properties, etc) and entities (e.g., places like "California") in the query, and then responds with a set of charts chosen based on the query shape (ranking, etc) and data existence constraints.

The custom instance uses a local open-source Python ML library, Sentence Transformers model, from [https://huggingface.co/sentence-transformers](https://huggingface.co/sentence-transformers), and does not use LLM fallback.

When you load data into a custom instance, the Data Commons NL server generates embeddings for both the main Data Commons data, and for your custom data, based on the statistical variables and search descriptions you have defined in your configuration. When a query comes in, the server generates equivalent embeddings, and the variables are assigned a relevance score based on cosine similarity.

File: custom_dc/faq.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+## Natural language processing
+
+### How does the natural language interface work?
+
+The NL interface uses an open-source Python ML library, Sentence Transformers model, from [https://huggingface.co/sentence-transformers](https://huggingface.co/sentence-transformers). When you load data into a custom instance, the Data Commons NL server generates embeddings for both the main Data Commons data, and for your custom data, based on the statistical variables and search descriptions you have defined in your configuration. When a query comes in, the server generates equivalent embeddings, and the variables are assigned a relevance score based on cosine similarity. 
+
+### Does the model use any Google technologies, such as Vertex AI?
+
+No. At this time, there is no plan to use any Google-specific ML technologies for NL processing in custom Data Commons. All models are open-source.
```
Comment (pradh) [inline custom_dc/faq.md:34]: While the base Data Commons uses Vertex AI, the custom instance uses open-source ML technologies only.

File: custom_dc/faq.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+### Where does the ML model run and where are embeddings stored?
+
+The ML model runs entirely on your custom Data Commons instance, inside the Docker image. It does not use any Google-hosted systems, and data is never leaked to the main Data Commons. If a natural-language query requires data to be joined from the main Data Commons site, the custom site will use the embeddings that are locally generated before making the call to the main Data Commons to fetch the data.
+
+### Does the model use feedback from user behavior to adjust scoring?
+
+No. At this time, there is no plan to enable user feedback mechanisms for custom Data Commons.
```
Comment (pradh) [inline custom_dc/faq.md:42]: However, users have the ability to improve their query quality by improving their search descriptions.

**Review by pradh — COMMENTED** [review]
Sorry for the delay!

File: custom_dc/faq.md (hunk 2 above)
Comment (kmoscoe) [reply custom_dc/faq.md:30]: Done, thank you!

**Review by pradh — APPROVED** [review]
Thanks for the edits!

**Pull request merged**